MetadataJsonPrinter.py

```diff
@@ -0,0 +1,69 @@
+# The purpose of this module is to return the metadata information in a json format or to get a specific part of information of the metadata.
+# we can get the metadata as a json format by calling the function getMetadata
+# additionally we can get any specific information of the metadata by calling the method getSpecific and passing the name of the value you want to get.
+# Example getSpecific('Name') return Checker_Framework.
+
+import json
+def getMetaData():
```

Comment:
change function name to follow google python style. Also, is this function specific to rise4fun? If so, please reflect it in function name. If not, please make a function documentation to describe what meta data this function would return.